AnalysisDataFormats/SUSYBSMObjects/src/classes_def.xml

```diff
@@ -47,8 +47,9 @@
    <class name="edm::Wrapper<susybsm::HSCPDeDxInfoCollection>"/>
    <class name="edm::Wrapper<susybsm::HSCPDeDxInfoValueMap>"/>
 
-   <class name="susybsm::MuonSegment" ClassVersion="10">
+   <class name="susybsm::MuonSegment" ClassVersion="11">
     <version ClassVersion="10" checksum="3541168201"/>
+    <version ClassVersion="11" checksum="841063529"/>
```

Comment:
By quick look I'd think the `MuonSegment`'s  `DTRecSegment4DRef` and `CSCSegmentRef` members would be broken. The types should point to the new container types, but the `Ref`s `ProductID` would point to the data products read from the input file.

Reply:
Ah, I see now that ROOT's schema evolution outright fails for the Ref members.